rosbag2_compression/package.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
-<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
-<package format="2">
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
```

Comment:
Lets put these changes into a different PR since they're unrelated

Reply:
are we using any features for package format 3, or why this change in the first place?

Reply:
I don't think we are.

Reply:
@piraka9011 We've been discussing that multiple times in the last weeks, let's not bump to v3 unless we have a good reason to do so. This is disruptive for rosbag2 to use multiple versions of package.xml